app/assets/javascripts/Components/Result/text_viewer.jsx

```diff
@@ -131,7 +142,7 @@ export class TextViewer extends React.Component {
             </a>
           </div>
         </div>
-        <pre id={RAW_TEXT_DIV_ID} name={RAW_TEXT_DIV_ID} className={this.props.type}>
+        <pre name={preElementName} ref={this.raw_content} className={this.props.type}>
```

Comment:
Just to clarify.... the `SyntaxHighlighter.HighlightAll` looks for an element with the same "name" attribute as its first argument. The fact that we were giving this `pre` element the same "id" value was just a coincidence?

And we're not setting the id now because it's not needed?